templates/geosafe/analysis/create.html

```diff
@@ -133,33 +133,33 @@
         }
 
         .option .set .badge {
-            background-color: #50cf5c;
+            background-color: #1a6c38;
```

Comment:
making it darker green did not give a significant difference between not selected and selected layer IMO. cc @carina1000